doc/classes/Color.xml

```diff
@@ -221,14 +221,15 @@
 				Returns a new color from [code]rgba[/code], an HTML hexadecimal color string. [code]rgba[/code] is not case sensitive, and may be prefixed with a '#' character.
 				[code]rgba[/code] must be a valid three-digit or six-digit hexadecimal color string, and may contain an alpha channel value. If [code]rgba[/code] does not contain an alpha channel value, an alpha channel value of 1.0 is applied.
 				If [code]rgba[/code] is invalid a Color(0.0, 0.0, 0.0, 1.0) is returned.
+				[b]Note:[/b] This method is not implemented in C#, but the same functionaliy is provided in the class constructor.
```

Comment:
```suggestion
				[b]Note:[/b] This method is not implemented in C#, but the same functionality is provided in the class constructor.
```
Please also amend the commit message to be more explicit (e.g. like the PR title).

Reply:
Thanks for the typo correction